libs/sdk-bindings/build.rs

```diff
@@ -20,17 +19,15 @@ fn setup_x86_64_android_workaround() {
                 "Unsupported OS. You must use either Linux, MacOS or Windows to build the crate."
             ),
         };
-        let clang_version =
-            env::var("NDK_CLANG_VERSION").unwrap_or_else(|_| DEFAULT_CLANG_VERSION.to_owned());
-        let linux_x86_64_lib_dir = format!(
-            "toolchains/llvm/prebuilt/{build_os}-x86_64/lib/clang/{clang_version}/lib/linux/"
+        let linux_x86_64_lib_pattern = format!(
+            "{android_ndk_home}/toolchains/llvm/prebuilt/{build_os}-x86_64/lib*/clang/**/lib/linux/"
         );
-        let linkpath = format!("{android_ndk_home}/{linux_x86_64_lib_dir}");
-        if Path::new(&linkpath).exists() {
-            println!("cargo:rustc-link-search={android_ndk_home}/{linux_x86_64_lib_dir}");
-            println!("cargo:rustc-link-lib=static=clang_rt.builtins-x86_64-android");
-        } else {
-            panic!("Path {linkpath} not exists. Try setting the NDK_CLANG_VERSION environment variable.");
+        match glob(&linux_x86_64_lib_pattern).unwrap().last() {
```

Comment:
Looks good. I have a nit that may not be useful. This might not always take the latest clang version if multiple are available. (Is that even ever the case?). (Do we even need the latest version?). Anyway, v9.0.0 will be the 'last' if v10.0.0 is also available, because the results from the glob are sorted alphabetically.

Reply:
The clang version is in the double-digits, so that won't be a problem for a while.

Reply:
> The clang version is in the double-digits, so that won't be a problem for a while.

Yeah true only minor and patch versions maybe.